examples/edwards25519-hash/src/edwards25519-hash.rs

```diff
@@ -0,0 +1,532 @@
+// This crate implements hash to curve for the edwards25519 elliptic curve.
+// This specification follows edwards25519_XMD:SHA-512_ELL2_NU_ as specified in
+// section 8.5 of https://www.ietf.org/archive/id/draft-irtf-cfrg-hash-to-curve-13.html#name-suites-for-curve25519-and-e
```

Comment:
The draft is at [version 16](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hash-to-curve/) now.
Can you update the spec to make sure it is compatible with the most recent version?